modules/coreos-layering-configuring-on.adoc

```diff
@@ -38,7 +38,12 @@ You can test a `MachineOSBuild` object to make sure it builds correctly without
 
 You should not need to interact with these new objects or the `machine-os-builder` pod. However, you can use all of these resources for troubleshooting, if necessary.
 
-You need a separate `MachineOSConfig` CR for each machine config pool where you want to use a custom layered image.
+You need a separate `MachineOSConfig` CR for each machine config pool where you want to use a custom layered image. You can apply a custom layered image by using any machine config pool in your cluster, including the control plane, worker, or custom pool.
+
+[NOTE]
+====
+For single node OpenShift (SNO) clusters, you can use only the control plane machine config pool to apply a custom layered image.
```

Comment:
@cheesesashimi Is this all we need to cover this issue? It appears from [your comments](OCL work as-is on SNO) that "OCL work as-is on SNO" save for this one caveat.